Issue 1673 import parameters chemistry error

brosaplanella opened the pull request:
# Description

This allows importing parameters via chemistry that is in the local directory. I don't think this fixes importing parameters from other folders via chemistry. This is a much more complicated issue (I believe), but this fix should be enough for the workshop.

Fixes #1673 

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [x] Bug fix (non-breaking change which fixes an issue)


# Key checklist:

- [x] No style issues: `$ flake8`
- [x] All tests pass: `$ python run-tests.py --unit`
- [x] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [x] Code is commented, particularly in hard-to-understand areas
- [x] Tests added that prove fix is effective or that feature works

brosaplanella commented:
PS: we need to include this fix in the released version for the workshop, so not sure what is the right pipeline for this.

tinosulzer commented:
Let's make a patch release for this. Can you make a PR to main with version 21.8.1?

brosaplanella commented:
So, do I merge this to main first with the new version number and then we merge main to develop?

tinosulzer commented:
Yep, but don't merge this directly to main since it will also merge the other recent changes to develop. You need to make a new branch from main with just these changes. Might be possible with git cherry pick or just do manually

brosaplanella commented:
When we switched to calendar versioning we removed the patch number (it is just 21.08 now). Should I bring it in? Technically it is not a new version but just a patch to fix a bug mostly because of the workshop (otherwise we could wait until the next release).

tinosulzer commented:
Hmm ok yeah change it to 21.08.post2 then (in setup.py), and just add the line to changelog but don't make it a new section
